src/App.js

```diff
@@ -18,6 +53,24 @@ function App() {
           Learn React
         </a>
       </header>
+    <SearchBar searchFood={searchFood}/>
+    <AddFoodForm addNewFood={addNewFood}/> 
+            <h1>Foodlist</h1>
+            <div className='container'>
+          {foods.map((oneFood) => {
+        return (
+          <div> 
```

Comment:
Remember when we use map its always need to put the key property to highest component 
```suggestion
          <div key={oneFood.name}> 
```